src/Octoshift/CliContext.cs

```diff
@@ -0,0 +1,26 @@
+using System;
+
+namespace OctoshiftCLI
+{
+    public static class CliContext
+    {
+        private static string _rootCommand;
+        private static string _executingCommand;
+
+        public static string RootCommand
+        {
+            set => _rootCommand = _rootCommand is null
```

Comment:
I suppressed CodeQL warning here because of the following two reasons:
1. These `static` fields are supposed to be set only once and in the application's entry point.
2. We don't care about thread safely here since the fields are supposed to be set at the entry point once and after that the values cannot change.